app/src/main/java/org/mozilla/focus/utils/GifImageView.java

```diff
@@ -0,0 +1,100 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Source taken from https://stackoverflow.com/questions/42898968/adding-gif-in-android-studio
+
+package org.mozilla.focus.utils;
+
+import android.content.Context;
+import android.view.View;
+import java.io.InputStream;
+import android.graphics.Movie;
+import android.graphics.Canvas;
+import android.util.AttributeSet;
+import android.os.SystemClock;
+import android.net.Uri;
+import java.io.FileNotFoundException;
+import android.util.Log;
+
+public class GifImageView extends View {
```

Comment:
Did you test this on all the Fire devices in the office? Movie doesn't seem well supported according to other SO users: https://stackoverflow.com/questions/3660209/display-animated-gif#comment11621337_7772170 but I'm glad we're trying a built-in solution.